src/MarketingPageAcceptanceTestsSpecflow/Steps/AboutSolution/LastUpdatedDate.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using MarketingPageAcceptanceTestsSpecflow.Utils;
+using TechTalk.SpecFlow;
+
+namespace MarketingPageAcceptanceTestsSpecflow.Steps.AboutSolution
+{
+    [Binding]
+    public class LastUpdatedDate
+    {
+        private UITest _test;
+        private ScenarioContext _context;
```

Comment:
I think both these fields should have `readonly` applied, but that applies across the solution, so I might get those done in another PR